plugins/mcreator-core/datalists/particles.yaml

```diff
@@ -69,3 +69,14 @@
 - LANDING_OBSIDIAN_TEAR
 - REVERSE_PORTAL
 - WHITE_ASH
+- SNOWFLAKE
+- DRIPPING_DRIPSTONE_LAVA
+- FALLING_DRIPSTONE_LAVA
+- DRIPPING_DRIPSTONE_WATER
+- FALLING_DRIPSTONE_WATER
+- GLOW_SQUID_INK
+- GLOW
+- WAX_ON
+- WAX_OFF
+- ELECTRIC_SPARK
+- SCRAPE

```

Comment:
LIGHT, DUST, DUST_COLOR_TRANSITION, FISHING, VIBRATION, MYCELIUM, FALLING_SPORE_BLOSSOM, SPORE_BLOSSOM_AIR, SMALL_FLAME seem to be missing from this list, however I also couldn't find an equivalent of WATER_WAKE, SUSPENDED_DEPTH, SPELL_MOB, SPELL_MOB_AMBIENT, SNOW_SHOVEL in Minecraft. If you think any of these match, you don't need to add them, otherwise add the new ones.